recipe/meta.yaml

```diff
@@ -14,10 +14,10 @@ source:
     - collada-dom.patch
 
 build:
-  number: 16
+  number: 17
   skip: true  # [win and vc<14]
   run_exports:
-    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
```

Comment:
Why are you tightening the run-export? Do we expect ABI changes between patch versions?